### ISSUE TITLE
Upgrade to steal-tools @ 0.16.6

### ISSUE THREAD

**Issue by pmgmendes** [bitovi/documentjs#270]
Would it possible to upgrade documentjs to steal-tools @ 0.16.6? This is the first steal-tools version where [source-map git repo dependency](https://github.com/stealjs/source-map/archive/0.4.2-bitovi.1.tar.gz) was replaced by [bitovi-source-map](https://www.npmjs.com/package/bitovi-source-map) package available at the public npm registry. My current issue with that our CI infrastruture "usually" doesn't has access to the public internet making it difficult to resolve transitive dependencies that need to be fetch at e.g. GitHub.

**Comment by justinbmeyer** [comment]
Yeah, that would be great.

**Comment by pmgmendes** [comment]
I've forked documenjs to check the impact of the requested change - https://github.com/pmgmendes/documentjs/commit/a0ea9188e6f58e1ec151956a41d19b3eb15c8a0a. 
I'm unable to do a local build due an unrelated issue so I've set up a build in Travis and the outcome its pretty much the same as current documentjs master, green for node 0.10 and 0.12, red for latest node - https://travis-ci.org/pmgmendes/documentjs/builds/222758343.

How should I proceed to make this change available?
